onadata/libs/models/sorting.py

```diff
@@ -38,10 +44,12 @@ def json_order_by(sort_list):
     return u""
 
 
-def json_order_by_params(sort_list):
+def json_order_by_params(sort_list, none_json_fields: Dict = {}):
     params = []
 
     for field in sort_list:
-        params.append(field.lstrip('-'))
+        field = field.lstrip('-')
+        if field not in none_json_fields.keys():
+            params.append(field.lstrip('-'))
```

Comment:
Should we add or update the unit tests for the `json_order_by_params()` function?

Reply:
Added in the latest commits